modules/yieldlabBidAdapter.js

```diff
@@ -303,6 +304,21 @@ function createUserIdString(eids) {
   return str.join(',')
 }
 
+/**
+ * Creates a string from an array of eids with ID provider and atype if atype exists
+ * @param {Array} eids
+ * @returns {String} idprovider:atype,idprovider2:atype2,...
+ */
+function createUserIdAtypesString(eids) {
+  const str = []
+  for (let i = 0; i < eids.length; i++) {
+    if (eids[i].uids[0].atype) {
+      str.push(eids[i].source + ':' + eids[i].uids[0].atype)
+    }
+  }
+  return str.join(',')
```

Comment:
Use semicolons please

Reply:
Hi @patmmccann,

we have not been using semicolons anywhere in the file so far. Also running `gulp lint` does not print any issues.

Would it make sense to add that kind of stuff to the linting rules and thus enforce consistency everywhere?

Reply:
we've attempted to do so, but fixing all the issues is a work in progress

https://github.com/prebid/Prebid.js/pull/8846

Reply:
That sounds reasonable. I have created #9386 to generally fix that for our adapter and will add semi colons to this PR as well.